.ci/setup_env.sh

```diff
@@ -70,6 +70,7 @@ kong-ngx-build \
     --kong-nginx-module $KONG_NGINX_MODULE_BRANCH \
     --luarocks $LUAROCKS \
     --openssl $OPENSSL \
+    --debug \
```

Comment:
Is this a leftover?

Reply:
This is needed. The test case use the debug level information to determine the successfulness of the call (we should use debug build for CI tests anyway as that enables OpenResty built-in assertions).